py4web/utils/form.py

```diff
@@ -303,7 +303,7 @@ def FormStyleBulma(table, vars, errors, readonly, deletable):
         "input[type=radio]": "radio",
         "input[type=checkbox]": "checkbox",
         "input[type=submit]": "button",
-        "input[type=password]": "password",
+        "input[type=password]": "input password",
```

Comment:
This would be nice to be made as a separate PR to master since this one will be pushing to grid.

Reply:
I agree but no need to slow down now. For the future we should follow this better practice.